html/semantics/interactive-elements/the-dialog-element/dialog-requestclose.html

```diff
@@ -136,7 +141,7 @@
       },`requestClose avoids abuse prevention logic ${testDescription}`);
 
       promise_test(async (t) => {
-        await setup(t,closedby);
+        setup(t,closedby);
```

Comment:
Does this test even need to be a promise_test anymore?

Reply:
It probably doesn't but I don't think it does harm to leave them all as `promise_test` to simplify later refactors? I kind of wish `test` handled promises too :sweat_smile: